### DISCUSSION TITLE
"PROCESS_LIST = GenEnsProd" not working

### DISCUSSION THREAD

**Discussion by BinbinZhou-NOAA** [dtcenter/METplus#1265] [Configuration] [answered]
I run METplus conf in which PROCESS_LIST = GenEnseProd
but failed. The Warning message is saying

 WARNING: PROCESS_LIST item GenEnsProd may be invalid.
11/15 00:35:15.931 metplus.GenEnsProd (met_util.py:181) ERROR: Fatal error occurred 

Not for sure the warning is the cause of the failure 

I used  the MET version  10.1.0-beta3 and the METplus version  4.1.0-beta3 

Binbin

**Comment by georgemccabe** [comment, marked as answer]
It looks like you have a typo in the value for PROCESS_LIST. It should be GenEnsProd, not GenEns**e**Prod.

However, the METplus wrapper for this tool is not available in 4.1.0-beta3.
See https://github.com/dtcenter/METplus/discussions/1258#discussioncomment-1633923

> While the MET tool gen_ens_prod was added in the MET 10.1.0-beta3 release, the METplus wrapper was added after this release. **The wrapper will be available in the METplus 4.1.0-beta4 release which is scheduled for this Monday**.

**Reply by BinbinZhou-NOAA** [reply]
Thanks!

Binbin
